src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age !== undefined ? (
```

Comment:
there is no need to do check for `undefined`.
you can leave just `age`...the variable will be automatically converted to a boolean value, so if its `true` than...

also in oreder to write less code, you can use `&&` operator instead of ternary conditional  operator

```{age && (```